src/components/dls/Button/Button.module.scss

```diff
@@ -313,7 +313,8 @@
 }
 
 // when button used as Popover's trigger, add style when Popover is open
-[data-state="open"] > button > .base {
-  background-color: var(--color-primary-medium);
-  @include utility.lighten-background-color;
-}
+// TODO: removing this for now, buggy. Need to a better implementation
```

Comment:
Not sure what's happening here

Reply:
<img width="351" alt="image" src="https://user-images.githubusercontent.com/12745166/146863851-2fa858ee-f018-4c2f-994d-17703568889b.png">

this is related to this nice to have effect (see the globe icon), when the popover menu is open.

will need to do a follow up PR to fix this.